blockchain/stake/go.mod

```diff
@@ -9,6 +9,14 @@ require (
 	github.com/decred/dcrd/txscript v1.0.1
 	github.com/decred/dcrd/wire v1.1.0
 	github.com/decred/slog v1.0.0
+	github.com/fsnotify/fsnotify v1.4.7 // indirect
+	github.com/golang/protobuf v1.1.0 // indirect
+	github.com/hpcloud/tail v1.0.0 // indirect
+	github.com/kr/pretty v0.1.0 // indirect
+	golang.org/x/sync v0.0.0-20180314180146-1d60e4601c6f // indirect
+	golang.org/x/sys v0.0.0-20180816055513-1c9583448a9c // indirect
+	gopkg.in/fsnotify.v1 v1.4.7 // indirect
+	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
```

Comment:
These additions confused me for a while, until I realized they are test dependencies.  Looks correct though, and I verified these versions match the ones it resolves for me (no update hiding here).